wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -35,6 +35,8 @@
                               {% if message.level_tag == "error" %}
                                 {# There is no error icon, use warning icon instead #}
                                 {% icon name="warning" class_name="messages-icon" %}
+                              {% elif message.extra_tags == "workflow" %}
```

Comment:
We also need a corresponding special case for the 'lock' / 'unlock' icon, so I'll revise this to move the icon-picking logic into a new template tag rather than letting this if/else sequence get out of control. (It looks like the `{% message_tags %}` tag is also 99% redundant now that we're no longer handling the icons through CSS.)